core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -642,7 +642,7 @@ void processMeasurementEvent(WriterJob job) {
                         }
 
                         case NewLineProtoParser.ENTITY_TYPE_INTEGER: {
-                            final int colType = writer.getMetadata().getColumnType(colIndex);
+                            final int colType = ColumnType.tagOf(writer.getMetadata().getColumnType(colIndex));
```

Comment:
tagOf unnecessary

Reply:
We will get to it when doing ILP, not obvious